src/routes/contact.js

```diff
@@ -153,4 +149,42 @@ router.delete('/:contact_id', async (req, res) => {
 	return utils.response(res, code, message);
 });
 
+// links contact with list of entities
+router.post('/link/:contact_id', async (req, res) => {
+	//todo lots of validation to validate what is passed to the endpoint
+	//todo util function to check if relationship already exists?
+	let code;
+	let message;
+	try {
+		const contact = await req.context.models.Contact.findOne({
+			where: {
```

Comment:
for these 1-attribute objects, IMO it looks better to avoid the newlines, but just a thought